cpp/data/frameProcessor/src/XspressProcessPlugin.cpp

```diff
@@ -496,88 +489,115 @@ void XspressProcessPlugin::process_frame(boost::shared_ptr <Frame> frame)
         // This must be offset according to the rank and number of processes
         uint32_t push_frame_id = ((frame_id / frames_per_block_) * concurrent_processes_) + concurrent_rank_;
         FrameMetaData mca_metadata(push_frame_id, ss.str(), raw_32bit, "", mca_dims);
-        boost::shared_ptr<Frame> mca_frame(new DataBlockFrame(mca_metadata, memory_ptrs_[index]->current_byte_size()));
-        memcpy(mca_frame->get_data_ptr(), memory_ptrs_[index]->get_data_ptr(), memory_ptrs_[index]->current_byte_size());
+        boost::shared_ptr<Frame> mca_frame(new DataBlockFrame(mca_metadata, memory_ptrs_[index]->size()));
+        memcpy(mca_frame->get_data_ptr(), memory_ptrs_[index]->get_data_ptr(), memory_ptrs_[index]->size());
         // Set the chunking size
-        mca_frame->set_outer_chunk_size((int)memory_ptrs_[index]->frames());
+        mca_frame->set_outer_chunk_size(frames_per_block_);
```

Comment:
This isn't formatting.

Reply:
So, I think this is actually an error from git. I checked this part of the code and this is an if/else inside this [loop](https://github.com/DiamondLightSource/xspress-detector/blob/main/cpp/data/frameProcessor/src/XspressProcessPlugin.cpp#L442) and it seems like git got confused with all the formatting changes and is reporting that what's inside the **else** statement got changed, when in reality this is just the **if** statement that got moved around when everything was formatted.